### PR TITLE
Wizard recipe: racket-v8.0.0

### PR DESCRIPTION
**Pull request by miguelraz**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: racket
* Version: v8.0.0

@staticfloat please review and merge.

### CODE REVIEW

**Comment by miguelraz** [comment]
Build fails because it can't find `racket` and `raco`.
I though that placing them in the `${prefix}/` folder would solve this issue, alas nope.

File: R/racket/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+sources = [
+    GitSource("https://github.com/racket/racket.git", "8c173392a1fc5f7a0023b6fc3c31c9b28d9cf474")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd racket/
+make -j ${nproc} base
+cp racket/bin/* ${prefix}/
```
Comment (giordano) [inline R/racket/build_tarballs.jl:18]:
```suggestion
cp -r racket/bin ${prefix}/.
```

**Comment by miguelraz** [comment]
Hmmm should consider adding this flag 

https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=10181&view=logs&j=07c0a4ad-0a29-5ed8-e8a1-57568fd8ae4d&t=1bb74008-223c-567a-d113-c5eddafe62ba&l=138

**Comment by miguelraz** [comment]
2 survivors! 🎉

**Comment by giordano** [comment]
https://docs.binarybuilder.org/stable/build_tips/#Autoconfigure-builds but not sure how, since you don't call `configure` directly

**Comment by miguelraz** [comment]
I tried something, let's see if that works.

**Comment by miguelraz** [comment]
It did not.

**Comment by miguelraz** [comment]
I have no idea why there's no x86_64_linux_gnu buildbot now.

**Comment by giordano** [comment]
Because to save time and resource we don't rebuild for successful platform.

**Comment by miguelraz** [comment]
Feel free to merge.

**Comment by giordano** [comment]
I can't if there are unsuccessful platforms :slightly_smiling_face:

**Comment by miguelraz** [comment]
Ah, I thought I had freedom to say "this build recipe won't support all these Linux platforms"

**Comment by giordano** [comment]
You have the freedom, but you must disable the failing platforms.  CI must be successful in order to be able to merge.  This is enforced in GitHub settings of the repository, we can't really merge if something is failing.

**Comment by miguelraz** [comment]
Ach so. Failing platforms disabled.

**Comment by miguelraz** [comment]
All green now @giordano 🎉

**Pull request merged**